components/src/dynamo/vllm/args.py

```diff
@@ -173,6 +176,11 @@ def parse_args() -> Config:
     config.tool_call_parser = args.dyn_tool_call_parser
     config.reasoning_parser = args.dyn_reasoning_parser
     config.custom_jinja_template = args.custom_jinja_template
+    config.max_model_len = engine_args.max_model_len
```

Comment:
Isn't this arg already configurable from vLLM CLI args? https://docs.vllm.ai/en/latest/configuration/engine_args.html#-max-model-len

```
python -m dynamo.vllm --max-model-len 1234 ...
```

Why does it need to be added to the Config object? Can you expand more on what this unlocks for FT testing that wasn't already available?